.env.example

```
@@ -1,6 +1,7 @@
 STANDARD_DOMAIN_LIST = "fxtwitter.com,fixupx.com,twittpr.com"
 STANDARD_BSKY_DOMAIN_LIST = "fxbsky.app,canary.fxbsky.app"
 STANDARD_TIKTOK_DOMAIN_LIST = "fixtok.wuff.gay,wuff.gay,dxtiktok.com,cocktiktok.com"
+STANDARD_INSTAGRAM_DOMAIN_LIST = "67instagram.com"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)